cmd/updatebinary.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"go.keploy.io/server/pkg/service/updateBinary" // Import the updateBinary package
+	"go.uber.org/zap"
+)
+
+// NewCmdUpdateBinary initializes a new command to update the Keploy binary file.
+func NewCmdUpdateBinary(logger *zap.Logger) *UpdateBinary {
+	updater := updateBinary.NewUpdater(logger)
+	return &UpdateBinary{
+		updater: updater,
+		logger:  logger,
+	}
+}
+
+// UpdateBinary holds the updater instance for managing binary updates.
+
+type UpdateBinary struct {
+	updater updateBinary.Updater
+	logger  *zap.Logger
+}
+
+// GetCmd retrieves the command to update the Keploy binary file.
+
```

Comment:
please remove this empty line. generally, there should not be a new line space between functions and their comments